app-backend/app/src/main/scala/toolrun/package.scala

```diff
@@ -0,0 +1,7 @@
+package com.azavea.rf
+
+import com.azavea.rf.datamodel.{PaginatedResponse, ToolRun}
+
+package object toolrun extends RfJsonProtocols {
+    implicit val paginatedToolRunFormat = jsonFormat6(PaginatedResponse[ToolRun])
```

Comment:
Was this necessary / was scala complaining? `PaginatedResponse[T]` should be serializable as long as there's an implicit in scope for serializing `T`